subsys/bluetooth/controller/Kconfig

```diff
@@ -112,6 +112,22 @@ config BLUETOOTH_CONTROLLER_LE_PING
 	help
 	  Enable support for Bluetooth v4.1 LE Ping feature in the Controller.
 
+config BLUETOOTH_CONTROLLER_PRIVACY
+	bool "LE Controller-based Privacy"
+	default y
+	select BLUETOOTH_RPA
+	help
+	  Enable support for Bluetooth v4.2 LE Controller-based Privacy feature in the Controller.
+
+config BLUETOOTH_CONTROLLER_RL_SIZE
+	prompt "LE Controller-based Privacy Resolving List size"
+	depends on BLUETOOTH_CONTROLLER_PRIVACY
+	int
+	default 8
+	range 1 8
```

Comment:
This is a bit of a long story. The problem comes with a hardware limitation for nRF5x. I can limit the range to 1 8 only when using nRF5x which would make more sense. The issue is that the hardware whitelist in the nRF5x devices is only 8 items long. I need this whitelist to avoid a loop in the ISR, which would make support for privacy on nRF51 impossible because of hard realtime restrictions. In the future we can have 2 implementations, a hardware one for nRF51 and a software one for nRF52 and onwards which allows for a bigger resolving list and whitelist.